app/components/login_button_component.rb

```diff
@@ -0,0 +1,60 @@
+class LoginButtonComponent < BaseComponent
+  attr_reader :action, :big, :color, :width, :height, :logo_path, :tag_options
+
+  alias_method :big?, :big
+
+  DEFAULT_HEIGHT = 16
+  DEFAULT_WIDTH = 121
+  BIG_HEIGHT = 20
+  BIG_WIDTH = 152
+
+  def initialize(
+    action: ->(**tag_options, &block) { button_tag(**tag_options, &block) },
+    big: false,
+    color: "light blue",
+    **tag_options
+  )
+    @action = action
+    @big = big
+    @color = color
+    @tag_options = tag_options
+  end
+
+  def logo_path
+    return "logo-white.svg" if color == "dark blue"
+    "logo.svg"
+  end
+
+  def width
+    return BIG_WIDTH if big?
```

Comment:
Can we avoid setting an explicit width and rely on padding and natural button width instead? For example, how would this work if the button text were translated to Spanish? ([example](https://login.gov/es/), see "Inicie sesión con Login.gov")